app/services/contract_generator.rb

```diff
@@ -19,10 +19,17 @@ def initialize(offers)
       @parser = TemplateParser.new(templates, @offer)
       header_end = set_header(0.7, 0.5, @parser.get_data("header"))
```

Comment:
OMG. paul gries would say you should give 0.7 a name. HEADER_OFFSET_PINK_TUESDAY_INCHES or something